sdk/storage/AzureStorageBlob/Source/DataStructures/TransferCollection.swift

```diff
@@ -47,7 +47,7 @@ public struct TransferCollection {
     }
 
     /// Remove all transfers in this `TransferCollection` from the database. All currently active transfers will be
-    /// cancelled.
+    /// canceled.
```

Comment:
Never been quite sure on this, but looking it up online suggests that American english uses one `l` for this word (British english uses two), but two are always used for "cancellation". 🙃

Reply:
I looked up the same and that's why I changed it 😆